src/cc2olx/models.py

```diff
@@ -535,25 +541,25 @@ def parse_resource(self, node):
             data['children'] = children
         return data
 
-    def parse_file(self, node):
-        href = node.get('href')
+    def _parse_file(self, node):
+        href = node.get("href")
```

Comment:
Similar to my other comment, please avoid syntactic changes (that don't violate the style guide), if you can avoid it.

Especially here, because we still use single quotes elsewhere in this file (eg: we didn't change _all_ of them, just _some_), it creates more churn to review.